src/aleph/vm/orchestrator/resources.py

```diff
@@ -134,6 +135,8 @@ async def about_system_usage(request: web.Request):
     """Public endpoint to expose information about the system usage."""
     period_start = datetime.now(timezone.utc).replace(second=0, microsecond=0)
     machine_properties = get_machine_properties()
+    # Refresh and get latest settings aggregate
+    await update_aggregate_settings()
```

Comment:
instead of refreshing the aggregate every time this page , we should be a timeout period and only refresh if the data is older than that period.

Reply:
I moved the code to service initialization, so now only executes it at boot time.